src/test/validation_tests.cpp

```diff
@@ -57,15 +57,31 @@ BOOST_AUTO_TEST_CASE(subsidy_limit_test)
 {
     const auto chainParams = CreateChainParams(*m_node.args, ChainType::MAIN);
     CAmount nSum = 0;
-    for (int nHeight = 0; nHeight < 14000000; nHeight += 1000) {
+    auto step = 1000;
```

Comment:
nit: Might consider it a constant:
```suggestion
    constexpr auto STEP = 1000;
```

Reply:
I would do this in prod code, here it just seems like noise.